### DISCUSSION TITLE
`mapped_locations` not taking effect

### DISCUSSION THREAD

**Discussion by Xiaorui-Huang** [JanDeDobbeleer/oh-my-posh#1768] [Troubleshoot] [answered]
I have tried the `mapped_locations` feature for path, but nothing seems to take into effect. I'm not sure if I'm using it wrong or there is an bug or compatibility issue. For reference, I am using windows terminal and pwsh 7/wsl.

                {
                    "background": "#FF9248",
                    "foreground": "#2d3436",
                    "powerline_symbol": "\ue0b0",
                    "mapped_locations_enabled": true,
                    "mapped_locations": {
                        "C:\\Users\\Leet\\GitHub": "\uF09B"
                    },
                    "properties": {
                        "home_icon": "\uf7db",
                        "folder_icon": "\uf07c",
                        "folder_separator_icon": "\ue0b1 ",
                        "mixed_threshold": 6,
                        "style": "mixed",
                        "template": "\u2800{{ .Path }}\u2800"
                    },
                    "style": "powerline",
                    "type": "path"
                },
<img src="https://user-images.githubusercontent.com/42769475/154344992-97578d40-28fb-4210-a750-ab8f0a61b41c.png"  width=400>

**Comment by lnu** [comment, marked as answer]
the `mapped_locations` section should be under `properties`
sample here https://ohmyposh.dev/docs/path
